From add91177e40881daa318c7a8af9f5e193522e430 Mon Sep 17 00:00:00 2001
From: Simeon Marijon <simeon.marijon@foss.st.com>
Date: Tue, 20 Jun 2023 14:36:56 +0200
Subject: [PATCH 0441/1141] arm64: defconfig: add stm32 backup registers
 support in addons fragment

Enable STM32 TAMP backup registers with fragment-02-defconfig-addons.config

Change-Id: Ied55fbb49ae40c176c1149abfa5622596daffdd4
Signed-off-by: Simeon Marijon <simeon.marijon@foss.st.com>
---
 arch/arm64/configs/fragment-02-defconfig-addons.config | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/fragment-02-defconfig-addons.config b/arch/arm64/configs/fragment-02-defconfig-addons.config
index 67c49b626bd2..9be9598778b1 100644
--- a/arch/arm64/configs/fragment-02-defconfig-addons.config
+++ b/arch/arm64/configs/fragment-02-defconfig-addons.config
@@ -50,6 +50,7 @@ CONFIG_NET_SCH_MULTIQ=m
 CONFIG_NET_SCH_PRIO=m
 CONFIG_NET_SCH_SKBPRIO=m
 CONFIG_NVMEM_STM32_ROMEM=y
+CONFIG_NVMEM_STM32_TAMP=y
 CONFIG_OF_OVERLAY=y
 CONFIG_PCIE_STM32=y
 CONFIG_PHY_STM32_COMBOPHY=y
-- 
2.39.2

